content/docs/welcome-guide/setup/setup-from-github.md

```diff
@@ -163,10 +163,61 @@ All contributions to the O3DE repo are expected to be staged in a fork before su
+
+1. **Pre-Built SDK Engine** - Choose this build type if you are primarily interested in project development and make infrequent or no changes to the engine source.
+
+{{< tabs name="Engine build instructions" >}}
+{{% tab name="Source Engine" %}}
+
+1. Create a package directory in a writeable location. The directory `C:\o3de-packages` is used in the examples that follow.
+
+    ```cmd
+    mkdir C:\o3de-packages
+    ```
+
+    This directory will be used by the O3DE package downloader to retrieve external libraries needed for the engine.
+
+1. Get the Python runtime. The Python runtime is not included in the GitHub repo. It is required by the `o3de` script, which you will use to perform common command line functions. This script requires **CMake** to be installed and accessible on your device's path. If you have not installed CMake, or get an error that CMake cannot be found when running the script, refer to the [O3DE System Requirements](../requirements) page for installation instructions.
+
+    Open a command prompt to the directory where you set up O3DE and run the `get_python` script.
+
+    ```cmd
+    python\get_python.bat
+    ```
+
+1. Use CMake to create the Visual Studio project for the engine. Supply the build directory, the Visual Studio generator, the path to the packages directory that you created, and any other project options. Paths can be absolute or relative. Alternatively, you can use the CMake GUI to complete this step.
+
+    ```cmd
+    cmake -B build/windows_vs2019 -G "Visual Studio 16" -DLY_3RDPARTY_PATH=C:\o3de-packages -DLY_UNITY_BUILD=ON
+    ```
+
+    There are several noteworthy custom definitions (`-D`) specified in the preceding command. All are optional but recommended in this example.
+
+    * `LY_3RDPARTY_PATH` : The path to the downloadable package directory, also known as the "third-party path". Do not use trailing slashes when specifying the path to the packages directory.
+    * `LY_UNITY_BUILD` : Unity builds are recommended in many cases for improved build performance. If you encounter a build error, disable unity builds to help debug the problem.
+
+1. (Optional) Use CMake to build the source engine. This step is optional because in the "source engine" build model, the engine is built inside of every _project_. To avoid building the engine twice if you plan on working with projects, you might want to wait until you learn how to create and build a project in a later section on creating projects. However, if you want to proceed and build the engine without a project, use the following CMake build command.
+
+    The `profile` build configuration is shown in this example.
+
+    ```cmd
+    cmake --build build/windows_vs2019 --target Editor --config profile -- /m
+    ```
+
+    The `/m` is a recommended build tool optimization, which tells the Microsoft compiler (MSVC) to use multiple threads during compilation to speed up build times.
```

Comment:
```suggestion
The `/m` is a recommended build tool optimization. It tells the Microsoft compiler (MSVC) to use multiple threads during compilation to speed up build times.
```

Reply:
The sentence here is relevant to the step above it. IMHO, the indentation here helps connect the sentence to that step.

Reply:
Good call. Ignore the indentation change here. The rest of the change is consistent with the same sentence earlier in this PR.